### ISSUE TITLE
Adding/removing Faction Ranks to /f access

### ISSUE THREAD

**Issue by Deiscorides** [MassiveCraft/Factions#975]
It would be awesome to be able to add/remove specific ranks to specific chunks. Examples:

/f access r rankname
/f access r Recruit
/f access r Member

With this you can have more range of what you want to do with the land. Like making a specific Recruit-only section. It would remove Member's access and grant Recruits access to it.

**Comment by Deiscorides** [comment]
Also, if "/f access r Officer" is used, it will revoke the use of "/f access" commands for Officers while they are in that chunk.

**Comment by BoFone** [comment]
Bump! 

Also, I found a problem with the /f access management.

When I set a chunk to be only accessible by one person under my faction, all Allies with build rights can still bypass that protection to edit that chunk even if all my own faction member don't have access to it. This doesn't make sense XD

**Comment by Deiscorides** [comment]
BoFone, check both "/f access v" and "/f perm s" to make sure you have your faction's access turned off and don't have Allies access turned on. If your faction is set to "True" it will still allow anyone who is a member of your faction to access that chunk even if one person is on the access list. It needs to be set to "False" using "/f access f yourfactionname". If "/f perm" for Allies is set so that they can build, it will bypass lots of stuff. It is better to prevent them building in "/f perm" and instead use "/f access f facname" to add or remove another faction to/from the "/f access v" list in a single chunk.

I hope that helps. Otherwise, I've no clue.

**Comment by BoFone** [comment]
yeah, thx for the support. I've set up the access management the correct way, which is only allowing one specific player to build in this chunk and excluding all the rest of my faction members. However, this situation is like your description in which I set Build perm for allies to true. Then I found out that this can bypass lots of stuff.

What I'm suggesting here is that a request to somehow let the /f access have the highest priority so that allies build perm cannot bypass it.
